pkg/apis/rook.io/v1alpha2/storage_test.go

```diff
@@ -48,7 +48,6 @@ func TestResolveNodeDefaultValues(t *testing.T) {
 func TestResolveNodeInherentFromCluster(t *testing.T) {
 	// a node with no properties defined should inherit them from the cluster storage spec
 	storageSpec := StorageScopeSpec{
-		Location: "root=default,row=a,rack=a2,chassis=a2a,host=a2a1",
```

Comment:
Made these changes to the test because we're ignoring cluster level information info.

Reply:
Could this impact other rook inhabitants?

Reply:
I see that only EdgeFS uses ResolveNode currently, and they do not use the Location field of v1alpha2.Node.